docs/docs/02-object_structure.md

```diff
@@ -12,8 +12,8 @@ must contain the following attributes:
 
 -   `isClass` - `boolean`: denotes whether the object to be drawn is a user-defined class (or a stack frame) or a built-in
     object. This has a default value of `false` and should be manually set to `true` only when drawing a class or stack frame.
--   `name` - `string`: denotes the type of the object to draw (if `isClass===true`, then this is the name of the
-    corresponding class or stack frame).
+-   `name` - `string`: The name of the class or stackframe to be drawn. Note that this attribute is only applicable if the object's 'isClass' attribute is true. Otherwise, this attribute can be excluded from the input.
```

Comment:
stackframe -> stack frame

for attribute names always use monospace font, indicated by backticks (`).